commitizen/commands/init.py

```diff
@@ -108,21 +199,90 @@ def _ask_tag_format(self, latest_tag) -> str:
             tag_format = r"v$version"
             is_correct_format = questionary.confirm(
                 f'Is "{tag_format}" the correct tag format?', style=self.cz.style
-            ).ask()
+            ).unsafe_ask()
 
         if not is_correct_format:
             tag_format = questionary.text(
                 'Please enter the correct version format: (default: "$version")',
                 style=self.cz.style,
-            ).ask()
+            ).unsafe_ask()
 
             if not tag_format:
                 tag_format = "$version"
         return tag_format
 
-    def _search_pre_commit(self) -> bool:
-        """Check whether pre-commit is installed"""
-        return shutil.which("pre-commit") is not None
+    def _ask_version_provider(self) -> str:
+        """Ask for setting: version_provider"""
+
+        OPTS = {
```

Comment:
Should we make it a variable in `version_provider.py`?

Reply:
I thought about it. But the thing is that the values are actually defined in pyproject.toml as part of `[tool.poetry.plugins."commitizen.provider"]`. Setting them in `version_provider` may create confusion, as they are only used by init. I think in the future we can try to read them from the plugins system (which I don't know yet how to do)